### DISCUSSION TITLE
Tech recommendations for contributing to the project

### DISCUSSION THREAD

**Discussion by mautas** [magicbug/Cloudlog#2900] [Q&A]
Hi, I'd like to contribute to Cloudlog. Even though I have a computer science degree, I've been very far from the programming experience for decades now. I know web development is quite complex, involving lots of different tools, I'd like to receive a few recommendations on what development tools to install and get aqcuainted with. I think I'd like to contribute to Cloudlog adding a new award for Argentina's provinces and federal district. Thanks in advance for any help, and Happy New Year to you all!!!

**Comment by magicbug** [comment]
Honestly not to many tools needed, I'm going to base this on running Windows so I use Visual Studio Code with Githubs Desktop Client, and then have a simple server setup for apache, PHP, MySQL using [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/about) but you can get a WAMP installer quite easily that can be used.

**Reply by mautas** [reply]
Thanks a lot for the prompt reply to my posting! I am also Windows based here, I'll give a look to that Windows Subsystem for Linux. It seems like Visual Studio Code is the way to go.

**Comment by AndreasK79** [comment]
@mautas I use XAMPP on Windows here, and the editor is Visual Studio Code. I use Git and PHP debugging in VS Code. Let us know if you need any help.

**Reply by mautas** [reply]
Thank you for your prompt answer! Peter also mentioned VS Code and since I am also windows based that seems like the way to go so I'll go ahead setting up my environment here. Your help offering is appreciated a lot! Thank you again!
